src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordClone.cs

```diff
@@ -150,6 +147,11 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
         // Note: this method was replicated in SymbolDisplayVisitor.FindValidCloneMethod
         internal static MethodSymbol? FindValidCloneMethod(TypeSymbol containingType, ref HashSet<DiagnosticInfo>? useSiteDiagnostics)
         {
+            if (containingType.IsObjectType())
```

Comment:
No difference. I think I used `.SpecialType` in some other location because it deals with a TSWA (which doesn't have `.IsObject()`, but has `.SpecialType`). Could also have done `.Type.IsObjectType()`.

---
In reply to: [468927914](https://github.com/dotnet/roslyn/pull/46616#discussion_r468927914) [](ancestors = 468927914)